indexer-app/src/cmr/indexer/data/concepts/service.clj

```diff
@@ -0,0 +1,33 @@
+(ns cmr.indexer.data.concepts.service
+  "Contains functions to parse and convert service and service association concepts."
+  (:require
+   [cmr.common.log :refer (debug info warn error)]
+   [cmr.indexer.data.concept-parser :as concept-parser]
+   [cmr.transmit.metadata-db :as mdb]))
```

Comment:
It seems odd that we added a metadata-db2 namespace, but we still write new code using the supposedly deprecated namespace.

Reply:
Agreed, I think we should write a refactoring ticket and schedule it to be worked. I don't worry too much about making another call to the function as this doesn't add to the refactoring work.